src/View/Widget/RadioWidget.php

```diff
@@ -32,20 +41,49 @@ public function render(array $data, ContextInterface $context)
             'inline' => false,
         ];
 
+        $templates = $this->_templates->get();
+        $customize = [];
+
         if ($data['inline']) {
-            $this->_templates->add(['radioContainer' => '{{content}}']);
+            $this->_inline = $data['inline'];
+            if ($templates['formGroup'] === '{{label}}{{input}}') {
+                $this->_templates->add(['radioFormGroup' => '{{label}}<div class="radio-inline-wrapper">{{input}}</div>']);
```

Comment:
Hardcoding template here means users can't customize it. Instead the template should be set in `FormHelper::$_templateSet` and switched in appropriate FormHelper method based on options.